src/Jackett.Common/Indexers/FileListApi.cs

```diff
@@ -0,0 +1,216 @@
+
+            TorznabCaps.SupportsImdbMovieSearch = true;
+
+            AddCategoryMapping(24, TorznabCatType.TVAnime, "Anime");
+            AddCategoryMapping(11, TorznabCatType.Audio, "Audio");
+            AddCategoryMapping(15, TorznabCatType.TV, "Desene");
+            AddCategoryMapping(18, TorznabCatType.Other, "Diverse");
+            AddCategoryMapping(16, TorznabCatType.Books, "Docs");
+            AddCategoryMapping(25, TorznabCatType.Movies3D, "Filme 3D");
+            AddCategoryMapping(6, TorznabCatType.MoviesHD, "Filme 4K");
+            AddCategoryMapping(26, TorznabCatType.MoviesBluRay, "Filme 4K Blu-Ray");
+            AddCategoryMapping(20, TorznabCatType.MoviesBluRay, "Filme Blu-Ray");
+            AddCategoryMapping(2, TorznabCatType.MoviesDVD, "Filme DVD");
+            AddCategoryMapping(3, TorznabCatType.MoviesForeign, "Filme DVD-RO");
+            AddCategoryMapping(4, TorznabCatType.MoviesHD, "Filme HD");
+            AddCategoryMapping(19, TorznabCatType.MoviesForeign, "Filme HD-RO");
+            AddCategoryMapping(1, TorznabCatType.MoviesSD, "Filme SD");
+            AddCategoryMapping(5, TorznabCatType.AudioLossless, "FLAC");
+            AddCategoryMapping(10, TorznabCatType.Console, "Jocuri Console");
+            AddCategoryMapping(9, TorznabCatType.PCGames, "Jocuri PC");
+            AddCategoryMapping(17, TorznabCatType.PC, "Linux");
+            AddCategoryMapping(22, TorznabCatType.PCPhoneOther, "Mobile");
+            AddCategoryMapping(8, TorznabCatType.PC, "Programe");
+            AddCategoryMapping(27, TorznabCatType.TVHD, "Seriale 4K");
+            AddCategoryMapping(21, TorznabCatType.TVHD, "Seriale HD");
+            AddCategoryMapping(23, TorznabCatType.TVSD, "Seriale SD");
+            AddCategoryMapping(13, TorznabCatType.TVSport, "Sport");
+            AddCategoryMapping(12, TorznabCatType.AudioVideo, "Videoclip");
+            AddCategoryMapping(7, TorznabCatType.XXX, "XXX");
+        }
+
+        public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken configJson)
+        {
+            LoadValuesFromJson(configJson);
+
+            var pingResponse = await RequestStringWithCookiesAndRetry(ApiUrl + "?action=latest-torrents", null, null, GetAuthHeader());
+
+            JArray json = null;
+            try
+            {
+                json = JArray.Parse(pingResponse.Content);
+            }
+            catch (Exception ex)
+            {
+                throw new ExceptionWithConfigData(ex.Message, configData);
+            }
+
+            IsConfigured = true;
+
+            SaveConfig();
+
+            return IndexerConfigurationStatus.Completed;
+        }
+
+        protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            var releases = new List<ReleaseInfo>();
+            var response = await CallProviderAsync(query);
+            releases.AddRange(TransformProviderRequest(query, response));
```

Comment:
`TransformProviderRequest` is only called here, please inline the function to avoid generating and copying multiple lists.